rmc/utils/constraint.py

```diff
@@ -206,37 +209,37 @@ def calculate_exp_from_mu(
     :param n_partitions: Number of desired partitions for temporary grouped context Table. Default is 5000.
     :return: Table annotated with per-variant expected counts.
     """
-    mu_expr = context_ht.mu_snp
+    mu_expr = possible_ht.mu_snp
     cov_corr_expr = get_coverage_correction_expr(
-        context_ht.coverage, context_ht.coverage_model
+        possible_ht.coverage, possible_ht.coverage_model
     )
 
     if locus_type == "X":
-        plateau_model = context_ht.plateau_x_models
+        plateau_model = possible_ht.plateau_x_models
     elif locus_type == "Y":
-        plateau_model = context_ht.plateau_y_models
+        plateau_model = possible_ht.plateau_y_models
     else:
-        plateau_model = context_ht.plateau_models
+        plateau_model = possible_ht.plateau_models
 
     agg_expr = {
         "mu": hl.agg.sum(mu_expr * cov_corr_expr),
         "mu_agg": hl.agg.sum(mu_expr),
     }
     agg_expr.update(
         compute_expected_variants(
-            ht=context_ht,
+            ht=possible_ht,
             plateau_models_expr=plateau_model,
             mu_expr=mu_expr,
             cov_corr_expr=cov_corr_expr,
-            possible_variants_expr=context_ht.possible_variants,
-            cpg_expr=context_ht.cpg,
+            possible_variants_expr=possible_ht.possible_variants,
+            cpg_expr=possible_ht.cpg,
         )
     )
 
     logger.info(
         "Grouping by %s and computing expected counts per grouping...", groupings
     )
-    group_ht = context_ht.group_by(*groupings).aggregate(**agg_expr)
+    group_ht = possible_ht.group_by(*groupings).aggregate(**agg_expr)
```

Comment:
this needs to be `possible_ht` because of the call to `compute_expected_variants` -- `possible_variants` needs to exist in the input Table (and it doesn't on the context table)